front-end/src/pages/PostDetailPage.tsx

```diff
@@ -28,10 +29,10 @@ export default function PostDetailPage(props: Props) {
   useEffect(() => {
     let getPromise;
     if (props.loggedInUser) {
-      getPromise = AxiosWrapper.get(`${process.env.REACT_APP_API_URL}/api/posts/${postId}`);
+      getPromise = AxiosWrapper.get(`${process.env.REACT_APP_API_URL}/api/posts/${postId}`, props.loggedInUser);
     }
     else {
-      getPromise = AxiosWrapper.get(`${process.env.REACT_APP_API_URL}/api/posts/${props.location.pathname}`,);
+      getPromise = axios.get(`${process.env.REACT_APP_API_URL}/api/posts/${props.location.pathname}`);
```

Comment:
trailing slash. This isn't using the AxiosWrapper since props.loggedInUser is undefined right?

Reply:
Yeah, that was the original reason I had this here, but I later changed the wrapper to properly handle undefined. I just refactored this to return to using the wrapper now.

Reply:
Not changing trailing slash since it's not needed here